packages/vscode-extension/src/commandsMap.ts

```diff
@@ -35,6 +36,11 @@ export const commandsInfoMap: CommandsInfoMapType = {
 		vscodeCommandName: 'Add Dependency References to the Manifest',
 		vscodeShortName: 'Add Dependencies',
 	},
+	comparefile: {
+		cliCommandName: 'file:import',
```

Comment:
why the command is file:import?

Reply:
There is no cli command for compare. We are actually calling file:import to import the file and then call vscode compare to compare the imported file to the file in the project.

Reply:
ok seems a bit strange but sure :)